### PR TITLE
fix(api): workflow v3 job name generate

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16854.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

**Review by sguiheux — APPROVED** [review]

File: sdk/workflowv3/convert.go [hunk 1]
```diff
@@ -239,12 +239,7 @@ func computeJobUniqueID(nodeName, stageName, actionName string, actionID int64)
 }
 
 func computeJobName(allJobs map[string]*Job, nodeName, stageName, actionName string, actionID int64) string {
-	jName := slug.Convert(actionName)
-	if _, ok := allJobs[jName]; !ok {
-		return jName
-	}
-
-	jName = slug.Convert(fmt.Sprintf("%s-%s", stageName, actionName))
+	jName := slug.Convert(fmt.Sprintf("%s-%s", nodeName, actionName))
```
Comment (fsamin) [inline sdk/workflowv3/convert.go:242]:
```suggestion
	jName := slug.Convert(nodeName +"-"+ actionName)
```

**Review by fsamin — CHANGES_REQUESTED** [review]

File: sdk/workflowv3/convert.go (hunk 1 above)
Comment (richardlt) [reply sdk/workflowv3/convert.go:242]: Will do this in a next PR

Comment (fsamin) [reply sdk/workflowv3/convert.go:242]: ok

**Review by fsamin — APPROVED** [review]

**Pull request merged**